components/datadog/agent/kubernetes.go

```diff
@@ -0,0 +1,55 @@
+package agent
+
+import (
+	"github.com/pulumi/pulumi-kubernetes/sdk/v3/go/kubernetes"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/components"
+	"github.com/DataDog/test-infra-definitions/components/datadog/kubernetesagentparams"
+)
+
+type KubernetesAgentOutput struct {
+	components.JSONImporter
+
+	AgentInstallNameLinux   string `json:"agentInstallNameLinux"`
+	AgentInstallNameWindows string `json:"agentInstallNameWindows"`
+}
+
+// DockerAgent is a Docker installer on a remote Host
```

Comment:
Fix comment

Reply:
Fixed in https://github.com/DataDog/test-infra-definitions/pull/618/commits/bd19bdbec4ebc19129d7c5341991d98df87d86b6